### PR TITLE
Add functional Wesleyan scraper

### PR DESCRIPTION
**Pull request by HarrisonGregg**
Straightforward scraper to use the public API.

### CODE REVIEW

File: osp_scraper/spiders/wesleyan.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+from ..spiders.CustomSpider import CustomSpider
+
+class WesleyanSpider(CustomSpider):
+    name = "wesleyan"
+
+    def start_requests(self):
+        # Terms are generated with JS on page, so logic is replicated here
+        terms = []
+        current_year = datetime.date.today().year % 100
```
Comment (jodizzle) [inline osp_scraper/spiders/wesleyan.py]: [Be cautious about running on computers with very out of date clocks.](https://github.com/opensyllabus/osp-scraper/pull/43#issuecomment-293644306)

Comment (HarrisonGregg) [inline osp_scraper/spiders/wesleyan.py]: I would have done it the same way I did with that other one, but I didn't think it was worth making an unnecessary request.

Comment (HarrisonGregg) [inline osp_scraper/spiders/wesleyan.py]: Do you have any reason to believe a computer would have an out of date clock?

Comment (jodizzle) [inline osp_scraper/spiders/wesleyan.py]: Oh no, I was mostly just referencing an older PR.  I think using `datetime` is okay, especially in this circumstance.

**Pull request merged**